### ISSUE TITLE
ERROR: Build failed with exception when execute lint 

### ISSUE THREAD

**Issue by VMadalin** [android/app-bundle-samples#5]
### Description
Execute the following command  `./gradlew lint` for analysing code cause the following error
Starting a Gradle Daemon (subsequent builds will be faster)

```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':features:assets:lint'.
> kotlin.collections.EmptyIterator cannot be cast to kotlin.collections.MutableIterator

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s

```

### Steps to reproduce

1.  Open terminal with project root path
2.  Execute the following command ./gradlew lint (on mac or linux) gradlew lint (on windows)

**Comment by Amud** [comment]
Hi @VMadalin : As mentioned by you, this seems a bug with built tool 3.5.0. But downgrading it to 3.4.2 seems not an option for someone looking for android dynamic features support. 
Instead of that, disabling lint check for Release Builds seems to be an easy option as follow : 
`lintOptions {
        abortOnError false
        checkReleaseBuilds false
    }`

**Comment by arifaizin** [comment]
fixed with update build tool to 3.6.2

**Comment by shivani805** [comment]
> ### Description
> Execute the following command `./gradlew lint` for analysing code cause the following error Starting a Gradle Daemon (subsequent builds will be faster)
> 
> ```
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Could not determine the dependencies of task ':features:assets:lint'.
> > kotlin.collections.EmptyIterator cannot be cast to kotlin.collections.MutableIterator
> 
> * Try:
> Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> BUILD FAILED in 9s
> ```
> 
> ### Steps to reproduce
> 1. Open terminal with project root path
> 2. Execute the following command ./gradlew lint (on mac or linux) gradlew lint (on windows)

getting an error "No such file or directory" when i run in project root path on mac
